CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 
+* [BUGFIX] TSDB: keep duplicate series records in checkpoints while their samples may still be present in the WAL. #16060
```

Comment:
I like to word entries so the intended audience (Prometheus admins) can read it and understand whether they might have been affected.
So, something like "fix unknown series errors and possible lost data from series that are removed after inactivity then re-appear before the next WAL checkpoint"

Reply:
updated in https://github.com/prometheus/prometheus/pull/16060/commits/c3a80df194bb4b407f86fd5f15868b3f608b085c